src/components/OSCALControlPart.js

```diff
@@ -92,7 +92,7 @@ export default function OSCALControlPart(props) {
             componentId={props.componentId}
             modificationAlters={props.modificationAlters}
             modificationSetParameters={props.modificationSetParameters}
-            key={part.id}
+            key={part.id ? part.id : part.name}
```

Comment:
```suggestion
            key={part.id ?? part.name}
```

Or we should just always use the `name`? It's required by the spec while `id` is optional

Reply:
It does not seem that the `rev4 catalog` actually follows the rule that names are unique. At the very least just using the name causes a key warning.

Reply:
@brian-easyd Can you clarify whether the intent of objects of type `token` (such as [Catalog > Control > Part > name](https://pages.nist.gov/OSCAL/reference/latest/catalog/json-reference/#/catalog/controls/parts/name)) are intended to be unique within a document or within a more limited scope?